Cargo.toml

```
@@ -35,6 +35,7 @@ syn = { version = "2.0.32", features = ['full', 'visit', 'extra-traits'] }
 to-syn-value = "0.1.1"
 to-syn-value_derive = "0.1.1"
 walkdir = "2"
+cbindgen = "0.26.0"
```

[`cbindgen` just (as of now 6 hours ago) released `0.27.0`](https://crates.io/crates/cbindgen) fixing some problems and removing some undesirable transitive dependencies. So an update to the current version would be good.

---

When updating you probably also want to set default-features to false i.e. 

```suggestion
cbindgen = { version = "0.27.0", default-features = false }
```